contracts/governance/src/governance.rs

```diff
@@ -10,29 +10,39 @@ use alloc::vec;
 use alloc::vec::Vec;
 use canonical_derive::Canon;
 use dusk_bls12_381::BlsScalar;
-use dusk_jubjub::GENERATOR_EXTENDED;
-use dusk_pki::PublicKey;
-use dusk_schnorr::Signature;
+use dusk_bls12_381_sign::{PublicKey as BlsPublicKey, Signature};
+use dusk_bytes::Serializable;
 
 use crate::collection::Collection;
 use crate::*;
 
-#[derive(Debug, Default, Clone, Canon)]
+#[derive(Debug, Clone, Default, Canon)]
 pub struct GovernanceContract {
     pub(crate) seeds: Collection<BlsScalar, ()>,
     pub(crate) balances: Collection<PublicKey, u64>,
     pub(crate) whitelist: Collection<PublicKey, ()>,
     pub(crate) paused: bool,
     pub(crate) total_supply: u64,
+    pub(crate) broker: BlsPublicKey,
+    pub(crate) authority: BlsPublicKey,
 }
 
+/// Use `GovernanceContract::default()` for instance.
 impl GovernanceContract {
-    /// Authority of the contract
-    ///
-    /// Will have to be defined in the constant space so the bytecode of the
-    /// contract will be changed as the authority does
-    pub const AUTHORITY: PublicKey =
-        PublicKey::from_raw_unchecked(GENERATOR_EXTENDED);
+    /// Update the authority public address of the Governance Contract
+    pub fn authority(&mut self, key: &[u8; 96]) -> Result<(), Error> {
```

Comment:
I would prefer to be more explicit in favour of `set_authority` or `update_authority`